msf_updates/src/position_pose_msf/position_pose_sensormanager.h

```diff
@@ -186,8 +186,15 @@ class PositionPoseSensorManager : public msf_core::MSF_SensorManagerROS<
     Eigen::Quaterniond yawq(cos(yawinit / 2), 0, 0, sin(yawinit / 2));
```

Comment:
Not your code, but could you fix this with the correct frames of reference. This should be the orientation of the imu wrt. world. I.e. q_wi_yaw